nexus/src/populate.rs

```diff
@@ -58,14 +59,26 @@ pub enum PopulateStatus {
     Failed(String),
 }
 
+/// Auxiliary data necessary to populate the database.
+pub struct PopulateArgs {
```

Comment:
Ack - I had the same thought when adding this. The choice of rack initialization could be the RSS call, or it could be on startup by Nexus.

Ultimately I figured it would be worth adding "as soon as it could possibly be known" - it's possible that Nexus is initialized a bit before the RSS handoff call is finished.